vault/logical_system_paths.go

```diff
@@ -64,38 +84,208 @@ func (b *SystemBackend) configPaths() []*framework.Path {
 		{
 			Pattern: "config/ui/headers/$",
 
-			Callbacks: map[logical.Operation]framework.OperationFunc{
-				logical.ListOperation: b.handleConfigUIHeadersList,
+			Operations: map[logical.Operation]framework.OperationHandler{
+				logical.ListOperation: &framework.PathOperation{
+					Callback: b.handleConfigUIHeadersList,
+					Summary:  "Return a list of configured UI headers.",
+				},
 			},
 
 			HelpDescription: strings.TrimSpace(sysHelp["config/ui/headers"][0]),
 			HelpSynopsis:    strings.TrimSpace(sysHelp["config/ui/headers"][1]),
 		},
 
 		{
-			Pattern:         "generate-root(/attempt)?$",
+			Pattern: "generate-root(/attempt)?$",
+			Fields: map[string]*framework.FieldSchema{
+				"otp": &framework.FieldSchema{
+					Type:        framework.TypeString,
+					Description: "Specifies a base64-encoded 16-byte value.",
```

Comment:
OTP might not be a 16 byte value anymore. It should be equal to the bytes of the token itself. Since token itself is not a UUID anymore and because of `b.` and `s.` syntaxes, it has changed.

Reply:
I've removed the field.